examples/proof-of-concept/rabbitmq/forgeops/kustomize/overlay/7.0/all/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+namespace: iot
+commonLabels:
+  app.kubernetes.io/name: "forgerock"
+resources:
+  - ../../../base/kustomizeConfig
+  - ../../../base/forgeops-secrets
+  - ../../../base/7.0/ds/cts
+  - ../../../base/7.0/ds/idrepo
+  - ../../../base/am
+  - ../../../base/amster
+  - ../../../base/idm
+  - ../../../base/end-user-ui
+  - ../../../base/login-ui
+  - ../../../base/admin-ui
+  - ../../../base/7.0/ingress
+  - ../../../base/ldif-importer
+  - ../../../base/ig
```

Comment:
It's probably worth removing IG seeing as we don't use it for this integration now.

Reply:
Removing IG makes skeffold fall over. Keeping it in for the time being.